.github/workflows/ai-test-generation.yml

```
@@ -0,0 +1,400 @@
+
+          Please generate tests for the files identified.
+          PROMPT_EOF
+
+          cat test-generation-prompt.md
+
+      # ======================================================================
+      # AI-Powered Test Generation (Multiple Options)
+      # ======================================================================
+
+      - name: 🎭 Generate Tests with Claude
+        if: ${{ github.event.inputs.ai_provider == 'claude' && secrets.ANTHROPIC_API_KEY != '' }}
```

actionlint (1.7.8)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)